website/source/docs/guides/monitoring-telegraf.html.md

```diff
@@ -0,0 +1,315 @@
+for data to be written to disk, a sign that Raft might be writing snapshots to disk too often.
+
+**What to look for:** if `cpu.iowait_cpu` greater than 10%.
+
+### Network activity - Bytes Recived
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `net.bytes_recv` | Bytes received on each network interface. |
+| `net.bytes_sent` | Bytes transmitted on each network interface. |
+
+**Why they're important:** A sudden spike in network traffic to Consul might be the result of a misconfigured
+application client causing too many requests to Consul. This is the raw data from the system, rather than a specific Consul metric.
+
+**What to look for:**
+Sudden large changes to the `net` metrics (greater than 50% deviation from baseline).
+
+**NOTE:** The `net` metrics are counters, so in order to calculate rates (such as bytes/second),
+you will need to apply a function such as [non_negative_difference][].
+
+### Network activity - RPC Count
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `consul.client.rpc` | Increments whenever a Consul agent in client mode makes an RPC request to a Consul server |
+| `consul.client.rpc.exceeded` | Increments whenever a Consul agent in client mode makes an RPC request to a Consul server gets rate limited by that agent's limits configuration.  |
+
+**Why they're important:** Similar to the Bytes recieved, these measurements indicate the current load created from a Consul agent, including when the load becomes high enough to be rate limited. A high RPC count, especially when rate-limited could imply a misconfigured Consul agent.
+
+**What to look for:**
+A high number (Need some help here :D )
+
+### Disk activity
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `diskio.read_bytes` | Bytes read from each block device. |
+| `diskio.write_bytes` | Bytes written to each block device. |
+
+**Why they're important:** If the Consul host is writing a lot of data to disk, such as under high volume workloads, there may be frequent major I/O spikes during leader elections. This is because under heavy load,
+Consul is checkpointing Raft snapshots to disk frequently.
+
+It may also be caused by Consul having debug/trace logging enabled in production, which can impact performance.
+
+Too much disk I/O can cause the rest of the system to slow down or become unavailable, as the kernel spends all its time waiting for I/O to complete.
+
+**What to look for:** Sudden large changes to the `diskio` metrics (greater than 50% deviation from baseline,
+or more than 3 standard deviations from baseline).
+
+**NOTE:** The `diskio` metrics are counters, so in order to calculate rates (such as bytes/second),
+you will need to apply a function such as [non_negative_difference][].
+
+[non_negative_difference]: https://docs.influxdata.com/influxdb/v1.5/query_language/functions/#non-negative-difference
```

Comment:
@banks Here 😄

Reply:
Oh cool. I forget about that link syntax. If you want to check it you can run `make website` from the website dir and see locally but I'd assume middleman will handle it just the same 😄